source/GlassView.Export/Models/AllocationInfo.cs

```diff
@@ -0,0 +1,12 @@
+using BenchmarkDotNet.Engines;
+using GlassView.Core.Models;
+
+namespace GlassView.Export.Models;
+
+public sealed class AllocationInfo(GcStats gCStats, Int64 allocatedBytes) : IAllocationInfo
```

Comment:
These direct mappings will disappear, as I plan to change the models to pure classes with getters and setters only....

Reply:
Done.